pkg/repository/clone_mirror.go

```diff
@@ -44,6 +44,9 @@ type CloneOptions struct {
 
 // CloneMirror clones a local mirror from the remote repository
 func CloneMirror(repo *V1Repository, components []string, targetDir string, selectedVersions []string, options CloneOptions) error {
+	fmt.Printf("start to clone mirror, targetDir is %s, selectedVersions are %s\n", targetDir, selectedVersions)
```

Comment:
Please use the upper case at the beginning of the log message and the `selectedVersions` is an array, you cannot use `%s` to format it.

Reply:
addressed

Reply:
addressed